src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/behandling/BehandlingsstatistikkService.kt

```diff
@@ -0,0 +1,205 @@
+        behandlingMetode: BehandlingMetode?,
+    ): BehandlingDVH {
+        val saksbehandling = behandlingService.hentSaksbehandling(behandlingId)
+        val sisteOppgaveForBehandling = finnSisteOppgaveForBehandlingen(behandlingId, oppgaveId)
+        val henvendelseTidspunkt = finnHenvendelsestidspunkt(saksbehandling)
+        val strengtFortroligAdresse = evaluerAdresseBeskyttelseStrengtFortrolig(saksbehandling.ident)
+        val saksbehandlerId = finnSaksbehandler(hendelse, totrinnskontrollService, gjeldendeSaksbehandler, behandlingId)
+        val beslutterId = utledBeslutterId(hendelse, behandlingId)
+        val relatertEksternBehandlingId: String? =
+            saksbehandling.forrigeBehandlingId?.let { behandlingService.hentSaksbehandling(it).eksternId.toString() }
+
+        return BehandlingDVH(
+            behandlingId = saksbehandling.eksternId.toString(),
+            behandlingUuid = behandlingId.toString(),
+            sakId = saksbehandling.eksternId.toString(),
+            aktorId = saksbehandling.ident,
+            registrertTid = saksbehandling.opprettetTid.atZone(europeOslo)
+                ?: henvendelseTidspunkt.atZone(europeOslo),
+            endretTid = hendelseTidspunkt.atZone(europeOslo),
+            tekniskTid = zonedNow(),
+            behandlingStatus = hendelse.name,
+            opprettetAv = maskerVerdiHvisStrengtFortrolig(
+                strengtFortroligAdresse,
+                saksbehandlerId,
+            ),
+            saksnummer = saksbehandling.eksternFagsakId.toString(),
+            mottattTid = henvendelseTidspunkt.atZone(europeOslo),
+            saksbehandler = maskerVerdiHvisStrengtFortrolig(
+                strengtFortroligAdresse,
+                saksbehandlerId,
+            ),
+            ansvarligEnhet = maskerVerdiHvisStrengtFortrolig(
+                strengtFortroligAdresse,
+                sisteOppgaveForBehandling?.tildeltEnhetsnr ?: MASKINELL_JOURNALFOERENDE_ENHET,
+            ),
+            behandlingMetode = behandlingMetode?.name ?: "MANUELL",
+            behandlingÅrsak = saksbehandling.årsak.name,
+            avsender = "NAV Tilleggstønader",
+            behandlingType = BehandlingType.valueOf(saksbehandling.type.name).toString(),
+            sakYtelse = saksbehandling.stønadstype.name,
+            behandlingResultat = saksbehandling.resultat.name,
+            resultatBegrunnelse = null, // TODO er fritekstfelt, og er ikke ønsket i statistikk før enum er implementert
+            ansvarligBeslutter =
+            if (!beslutterId.isNullOrEmpty()) {
+                maskerVerdiHvisStrengtFortrolig(
+                    strengtFortroligAdresse,
+                    beslutterId.toString(),
+                )
+            } else {
+                null
+            },
+            vedtakTid = if (Hendelse.VEDTATT == hendelse) {
+                hendelseTidspunkt.atZone(europeOslo)
+            } else {
+                null
+            },
```

Comment:
Kommer litt an på hvordan dvh bruker/tolker dette feltet. Men ville forventet at vedtakstidspunktet er tidspunktet saken blir belsuttet på? Er først da saken blir verdig vedtak, og også datoen fra dette vi bruker i brev.

Reply:
ut frå dokumentasjonen til saksstatestikk så er det dei legger i dette feltet : Tidspunktet for når vedtaket ble fattet - hvis saken ble vedtatt.